aap-14811 Added modules for section 4.5 (#812)
ansible/aap-docs: downstream/modules/aap-hardening/ref-keeping-everything-up-to-date.adoc

// Module included in the following assemblies:
// downstream/assemblies/assembly-aap-security-use-cases.adoc

[id="ref-keeping-everything-up-to-date_{context}"]

= Keeping everything up to date

[role="_abstract"]

> For some RHEL servers, such as a lab or other non-production systems, there may be a desire to install all available patches on a regular cadence.  The following example playbook might be used in a job template that is scheduled to run weekly, and will update the system with all of the latest RPMs.

Reviewer: ```suggestion
For some RHEL servers, such as a lab or other non-production systems, administrators might want to install all available patches on a regular cadence. 
The following example playbook can be used in a job template that is scheduled to run weekly. It updates the system with all of the latest RPMs.
```
